pinot-core/src/main/java/com/linkedin/pinot/core/realtime/impl/kafka/KafkaConnectionHandler.java

```diff
@@ -186,9 +198,9 @@ public ConnectingToBootstrapNode() {
     @Override
     public void process() {
       // Connect to a random bootstrap node
-      if (_simpleConsumer != null) {
+      if (_kafkaSimpleConsumer != null) {
```

Comment:
the package name qualifies simpleConsumer, and this is in a Kafka specific class, so renaming is not necessary

Reply:
removed unnecessary renaming